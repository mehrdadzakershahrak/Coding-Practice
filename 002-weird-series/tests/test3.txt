7 19 40
