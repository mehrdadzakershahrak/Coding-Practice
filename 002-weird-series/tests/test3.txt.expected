0
7
14
19
21
26
28
33
35
38
40
42
45
47
49
52
54
56
57
59
61
63
64
66
68
70
71
73
75
76
77
78
80
82
83
84
85
87
89
90
